hazelcast/src/main/java/com/hazelcast/client/impl/protocol/ClientMessageFramingHelper.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2008-2015, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.client.impl.protocol;
+
+import java.util.List;
+
+/**
+ * Chunks given client message to frames
+ */
+public interface ClientMessageFramingHelper {
```

Comment:
Now that you asked, I am not sure. :) . 
It was that way because we will implement framing/chunking later. 
I have a second idea. I will implement one complete helper class(without interface), which will get frame size. I will pass int max as default and later when we can implement the details of that class and pass different frame sizes. It will be more clear, ones I update the pr.